pkg/api/validation/validation.go

```diff
@@ -2024,6 +2024,14 @@ func ValidateHostAliases(hostAliases []api.HostAlias, fldPath *field.Path) field
 	return allErrs
 }
 
+func ValidatePriorityName(priorityName string, fldPath *field.Path) field.ErrorList {
```

Comment:
Please don't allow priorityName to be updated unless you are absolutely sure you are going to use it in 1.7.  If you aren't, leave it create only.  We can never tighten validation.